persist/postgres/contracts.go

```diff
@@ -498,6 +498,21 @@ func (s *Store) RejectPendingContracts(ctx context.Context, maxFormation time.Ti
 	})
 }
 
+// UpdateContractRenewedTo updates the renewed to ID of a contract to the provided one.
+func (tx *updateTx) UpdateContractRenewedTo(contractID types.FileContractID, renewedTo *types.FileContractID) error {
+	var value any
+	if renewedTo != nil {
+		value = sqlHash256(*renewedTo)
+	}
+	res, err := tx.tx.Exec(tx.ctx, `UPDATE contracts SET renewed_to = $1 WHERE contract_id = $2`, value, sqlHash256(contractID))
```

Comment:
You should be able to drop the if condition if you do the following

```suggestion
	res, err := tx.tx.Exec(tx.ctx, `UPDATE contracts SET renewed_to = $1 WHERE contract_id = $2`, (*sqlHash256)(renewedTo), sqlHash256(contractID))
```